scripts/setup-adapters.sh

```diff
@@ -54,33 +54,26 @@ function install_aws_deps {
 }
 
 function install_gcs-sdk-cpp {
-  # Install gcs dependencies
+  # Install gcs dependencies that cannot be installed via cmake's FetchContent
   # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md#required-libraries
 
-  # abseil-cpp
-  github_checkout abseil/abseil-cpp 20240116.1 --depth 1
-  sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h"
-  cmake_install -DBUILD_SHARED_LIBS=OFF \
-    -DABSL_BUILD_TESTING=OFF
-
   # crc32
   github_checkout google/crc32c 1.1.2 --depth 1
-  cmake_install -DBUILD_SHARED_LIBS=OFF \
+  cmake_install \
+    -DBUILD_SHARED_LIBS=OFF \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=OFF \
```

Comment:
Why this addition, also build shared is duplicated.

Reply:
Opsie! Too much copy-n-paste over the weekend.